src/vivarium/examples/boids/location.py

```diff
@@ -30,22 +31,21 @@ def columns_created(self) -> List[str]:
     #####################
 
     def setup(self, builder: Builder) -> None:
-        self.width = builder.configuration.location.width
-        self.height = builder.configuration.location.height
+        self.config = builder.configuration.location
 
     ########################
     # Event-driven methods #
     ########################
 
     def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         count = len(pop_data.index)
-        # Start clustered in the center with small random velocities
+        # Start randomly distributed, with random velocities
         new_population = pd.DataFrame(
             {
-                "x": self.width * (0.4 + 0.2 * np.random.random(count)),
-                "y": self.height * (0.4 + 0.2 * np.random.random(count)),
-                "vx": -0.5 + np.random.random(count),
-                "vy": -0.5 + np.random.random(count),
+                "x": self.config.width * np.random.random(count),
+                "y": self.config.height * np.random.random(count),
+                "vx": (1 - np.random.random(count) * 2) * self.config.max_velocity,
```

Comment:
Nit: I think this is more intuitive:
```suggestion
                "vx": ((2 * np.random.random(count)) - 1) * self.config.max_velocity,
```

Reply:
Agreed!